Description: Do not use urllib3 from requests
 Upstream code is using the embedded version of urllib3 from requests, but the
 reuqests package in Debian has "unvendorized" it, so the functions aren't
 present in requests.
 .
 This patch restores sanity and uses urllib3 directly.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: no
Last-Update: 2014-09-17

Index: python-glanceclient/glanceclient/common/http.py
===================================================================
--- python-glanceclient.orig/glanceclient/common/http.py	2014-12-23 16:23:23.000000000 +0800
+++ python-glanceclient/glanceclient/common/http.py	2014-12-23 16:26:47.000000000 +0800
@@ -20,7 +20,7 @@
 from oslo.utils import netutils
 import requests
 try:
-    from requests.packages.urllib3.exceptions import ProtocolError
+    from urllib3.exceptions import ProtocolError
 except ImportError:
     ProtocolError = requests.exceptions.ConnectionError
 import six
Index: python-glanceclient/tests/test_ssl.py
===================================================================
--- python-glanceclient.orig/tests/test_ssl.py	2014-12-23 16:23:23.000000000 +0800
+++ python-glanceclient/tests/test_ssl.py	2014-12-23 16:26:47.000000000 +0800
@@ -16,7 +16,10 @@
 import os
 
 from OpenSSL import crypto
-from requests.packages.urllib3 import poolmanager
+try:
+    from requests.packages.urllib3 import poolmanager
+except:
+    from urllib3 import poolmanager
 import testtools
 
 from glanceclient.common import http
